ui/tests/integration/components/pki/pki-role-form-test.js

```diff
@@ -111,8 +111,50 @@ module('Integration | Component | pki-role-form', function (hooks) {
     await click(SELECTORS.roleCreateButton);
   });
 
-  /* FUTURE TEST TODO:
-   * it should update role
-   * it should unload the record on cancel
-   */
+  test('it should rollback attributes or unload record on cancel', async function (assert) {
```

Comment:
I think it might be better to split these up into 2 tests rather than rendering twice.

Reply:
good suggestion, I'll go ahead and amend.